cdk/lib/mobile-save-for-later.ts

```diff
@@ -127,8 +129,41 @@ export class MobileSaveForLater extends GuStack {
       ],
     });
 
+    // N.B. we cannot use GuCertificate here as we deploy to eu-west-1 but the certificate must be created in us-east-1.
```

Comment:
Do you happen to know why we created the certificate in `us-east-1`?

I noticed that the certificate used in MAPI was deployed to `eu-west-1`.

Reply:
Thank you Jacob for the context!